notebooks/Zero_cost_Zero_time_Zero_shot_Financial_Sentiment_Analysis.ipynb

```diff
@@ -0,0 +1,2428 @@
+{
```

Comment:
<p>installs should probably all be at the very beginning</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/setfit/pull/497/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>